rclcpp/src/rclcpp/utilities.cpp

```diff
@@ -89,14 +93,11 @@ signal_handler(int signal_value)
 void
 rclcpp::utilities::init(int argc, char * argv[])
 {
-  (void)argc;
-  (void)argv;
   g_is_interrupted.store(false);
-  rmw_ret_t status = rmw_init();
```

Comment:
I believe `rcl_init` is being called one line below. I believe the problem is that `rcl_init` does not call `rmw_init`, not that `rclcpp` does not call `rmw_init`.